docs/operations.md

```diff
@@ -458,3 +459,102 @@ cluster is available in case of a disaster on the primary cluster.
 If you are using this strategy, we recommend you only support manual fail over from primary to secondary as
 automated strategies risk causing a [split brain](http://en.wikipedia.org/wiki/Split-brain_%28computing%29)
 problem.
+
+## Certificate update upon expiry
+
+In EventStoreDB, the certificates require updating when they have expired or are going to expire soon. The user has to follow the below steps to perform rolling certificate update:- 
+
+### Step 1: Generate new certificates
+
+The new certificates can be created in the same manner as you generated the existing certificates.
+You can also use the EventStore [es-gencert-cli](https://github.com/EventStore/es-gencert-cli) tool to generate the CA and node certificates. 
+You can also follow the [Configurator](https://configurator.eventstore.com/) to create commands for generating the certificates based on your cluster's configuration.
+
+### Step 2: Replace the certificates
+
+The next step is to replace the newly generated certificates in a subfolder and update the symlink to point it to the new certificates.
+
+### Linux OS
+
+We normally have a symlink that points to the existing certificates like:-
+
+ca -> /etc/eventstore/certs/ca
+node.crt -> /etc/eventstore/certs/node.crt
+node.key -> /etc/eventstore/certs/node.key
+
+Update the symlink so that the link points to the new certificates. For example:-
+
+ca -> /etc/eventstore/newCerts/ca
+node.crt -> /etc/eventstore/newCerts/node.crt
+node.key -> /etc/eventstore/newCerts/node.key
+
+In the above example we have the links in a folder at path /home/ubuntu/links
+
+
+### Step 3: Reload the configuration
```

Comment:
Just one other thing to add here - we can also trigger a certificate reload by sending a SIGHUP signal to the EventStore process (on Linux only). We should mention this here too.